libs/wingc/src/jsify.rs

```diff
@@ -411,6 +472,9 @@ impl<'a> JSifier<'a> {
 								ExprKind::Reference(Reference::InstanceMember { object, .. }) => {
 									self.jsify_expression(&object, ctx)
 								}
+                ExprKind::Reference(Reference::TypeMember { .. }) => {
```

Comment:
spaces instead of tabs. Also I think there's an issue here if the type name is under a namespace:
`MyModule.MyStruct.field`

Reply:
hmm I think the namespaces issue is going to be a lot more complex.. IE if I have the following two wing files
lib.w
```js
struct Person {
  name: str;
  age: num;
}
```

main.w
```js
bring "./lib.w" as lib;

struct Advisor extends lib.Person {
	empId: str;
}
```

Then when we make it to jsification the We end up with a parent that looks like this
```js
 UserDefinedType {
    root: Symbol {
        name: "lib",  // <=== this is a problem
        span: WingSpan {
```

I have some ideas, but I think I would prefer to introduce them in a follow-up PR since this one is already pretty large. If there arent any objections?

Reply:
ok, but maybe add a comment there or a failing test that should pass in the future. I think the issue isn't only related to bringing other .w files. We also generate namespaces when importing a JSII lib. And JSII lib's with what's known as `datatype` interfaces generate wing structs (under the brought namespace). So you can actually test this now. Trying something like
```wing
bring cloud;
let x = cloud.BucketProps.fromJson( { public: false } );
```

Reply:
I added a commented out test and tracking issue: https://github.com/winglang/wing/issues/3792